frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -1,14 +1,22 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
 import * as PropTypes from 'prop-types';
-import { FilterSidePanel, VerticalTabs } from 'patternfly-react-extensions';
+import {
+  FilterSidePanel,
+  FilterSidePanelCategory,
+  FilterSidePanelCategoryItem,
+  VerticalTabs,
+  VerticalTabsTab,
+} from '@patternfly/react-catalog-view-extension';
 import { FormControl } from 'patternfly-react';
```

Comment:
While you're in here, mind migrating`<FormControl>` to a plain HTML element so we can eliminate this patternfly-react dependency?

Reply:
I can give this a go. @jeff-phillips-18 raised an issue about how the catalog tiles may be better off with more programmatic description heights given how much the content can vary, so there may be some more changes here anyway as I work on that.

Reply:
Thanks, @rebeccaalpert.  It should be as simple as:
```javascript
              <input
                className="pf-c-form-control co-catalog-page__input"
                type="text"
                ref={(ref) => (this.filterByKeywordInput = ref)}
                placeholder="Filter by keyword..."
                value={activeFilters.keyword.value}
                onChange={(e) => this.onKeywordChange(e.target.value)}
```

Reply:
I have a PR up in Joe's branch for the input; should be in soonish.